frontend/src/components/workspaces/WorkspaceContainer/WorkspaceContainer.tsx

```diff
@@ -0,0 +1,131 @@
+import { FC, useState, useEffect } from 'react';
+import { UserSwitchOutlined, PlusOutlined } from '@ant-design/icons';
+import Button from 'antd-button-color';
+import { TemplatesTable } from '../Templates/TemplatesTable';
+import { TemplatesEmpty } from '../Templates/TemplatesEmpty';
+import Box from '../common/Box';
+
+export interface IWorkspaceContainerProps {
+  workspace: {
+    id: number;
+    title: string;
+    role: string;
```

Comment:
For the role specify the type `"user" | "manager"` . (also in all other components where you use the same approach)

Reply:
Changed applied [here](https://github.com/netgroup-polito/CrownLabs/pull/460/commits/77c8d7a732e466080c9e305ba922259e2ecdaa42)